content/cnbattle/drone-upx/index.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2016-01-01T00:00:00+00:00
+title: Drone Upx
```

Comment:
IMHO a better title would be simply `UPX`, it should be already clear enough that we are talking about Drone plugins.